### ISSUE TITLE
Bug: rain gives NULL mm

### ISSUE THREAD

**Issue by sandervandegeijn** [josecastroleon/pimatic-openweather#34]
Since upgrade to 0.9, the rain attribute always gives null mm. I do get updates for the other variables, haven't changed the config.
![capture](https://cloud.githubusercontent.com/assets/2643715/18997871/96aee6ce-8736-11e6-9676-7facfb762665.PNG)

**Comment by mwittig** [comment]
Can you please send me the config for the weather device?

**Comment by sandervandegeijn** [comment]
Sure:

```
{
  "id": "weatherCurrent",
  "class": "OpenWeatherDevice",
  "name": "Weer in Driel",
  "location": "Driel, NL",
  "units": "metric",
  "lang": "en",
  "timeout": 60000,
  "blacklist": []
},
```

**Comment by mwittig** [comment]
Thanks. The problem is that some weather stations do not provide rain data for your location. Strange enough a new case popped up: Instead if the rain property being absent (normal no data case), the rain property is present with an empty dictionary value. I'll fix error handling asap for this case. 

```
weather: 
   [ { id: 800,
       main: 'Clear',
       description: 'clear sky',
       icon: '01d',
       iconUrl: 'http://openweathermap.org/img/w/01d.png' } ],
  base: 'stations',
  main: 
   { temp: 17.92,
     pressure: 1015,
     humidity: 56,
     temp_min: 16.67,
     temp_max: 19.44 },
  wind: { speed: 6.02, deg: 257.5 },
  rain: {},
  clouds: { all: 0 },
  dt: 1475252164,
  sys: 
   { type: 3,
     id: 1472217966,
     message: 0.0064,
     country: 'NL',
     sunrise: 1475213858,
     sunset: 1475255651 },
  id: 2756591,
  name: 'Driel',
  cod: 200,
  list: [] } 
```

**Comment by mwittig** [comment]
published pimatic-openweather@0.9.1

**Comment by sandervandegeijn** [comment]
Fix confirmed, thanks for the fast response :)
